crates/composer_api/src/lib.rs

```diff
@@ -12,8 +12,57 @@ use std::{
 
 pub const DEFAULT_SERVER_ADDRESS: &str = "localhost:8888";
 
+/// Composer expects `EventMessage` as the incoming probe data.
 #[derive(Serialize, Deserialize, Debug)]
-pub enum Event {
+pub struct EventMessage {
```

Comment:
Thought: perhaps we can as well call this `Packet`? That's what it is now if we peel any abstraction layers, and as Jake puts it in our old Coding Primer:
> 4. **Abstraction is pain**
*Fight* the urge to abstract a problem into a general-purpose library. Wait until you *need* it. Abstraction layers carry a significant mental tax. Design your code so it takes the least amount of time to understand for a new reader.

Reply:
Renamed to `Packet`.

// It's a 3-yo coding primer that Jake helped put together. I've always wanted to get back to it to refine and adjust it to the current team. Maybe it's one of the first things that we can work together at tonari :)